data/core/dirwatch.lua

```diff
@@ -28,7 +28,7 @@ function dirwatch:scan(directory, bool)
 end
 
 -- Should be called on every directory in a subdirectory.
--- In windows, this is a no-op for anything underneath a top-level directory,
+-- On windows, this is a no-op for anything underneath a top-level directory,
```

Comment:
At this point I'd also capitalize Windows.

Reply:
agree, good catch.